modules/encrypt-browser/src/encrypt.ts

```diff
@@ -60,6 +62,10 @@ export async function encrypt (
   plaintext: Uint8Array,
   { suiteId, encryptionContext, frameLength = FRAME_LENGTH }: EncryptInput = {}
 ): Promise<EncryptResult> {
+
+  /* Precondition: The frameLength must be less than the maximum frame size for browser encryption. */
+  needs(frameLength > 0 && Maximum.FRAME_SIZE >= frameLength, 'frameLength out of bounds')
```

Comment:
Can we add those bounds to the error message?

Reply:
fixed, in both... :)